config/formats.js

```diff
@@ -206,6 +206,28 @@ let Formats = [
 			'Arceus', 'Reshiram', 'Zekrom', 'Kyurem', 'Xerneas', 'Yveltal', 'Solgaleo', 'Lunala', 'Necrozma-Dusk-Mane', 'Necrozma-Dawn-Wings',
 		],
 	},
+	{
+		name: "[Gen 7] Let's Go! Cup",
+		threads: [`&bullet; <a href="https://www.smogon.com/forums/threads/3639342/">Let&apos;s Go! Cup</a>`],
```

Comment:
There's no reason to escape the apostrophe, it should just be an ASCII apostrophe.